src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -3278,12 +3278,13 @@ private void getEEInfo(ref CORINFO_EE_INFO pEEInfoOut)
             // In debug, write some bogus data to the struct to ensure we have filled everything
             // properly.
             fixed (CORINFO_EE_INFO* tmp = &pEEInfoOut)
-                MemoryHelper.FillMemory((byte*)tmp, 0xcc, Marshal.SizeOf<CORINFO_EE_INFO>());
+                NativeMemory.Fill(tmp, (nuint)sizeof(CORINFO_EE_INFO), 0xcc);
 #endif
 
             int pointerSize = this.PointerSize;
 
             pEEInfoOut.inlinedCallFrameInfo.size = (uint)SizeOfPInvokeTransitionFrame;
+            pEEInfoOut.inlinedCallFrameInfo.sizeWithSecretStubArg = (uint)SizeOfPInvokeTransitionFrame + (uint)pointerSize;
```

Comment:
```suggestion
```
This should remain uninitialized. R2R and native AOT should not ever use the hidden stub arg.

Reply:
Are the inlined call frames used for R2R at all? If they are doesn't it mean that this change requires R2R version bump?

Reply:
This does not require R2R version bump.

R2R and NAOT use `ShouldUsePInvokeHelpers` that abstract away the PInvoke transition details. The JIT is just required to allocate opaque local of specified size and the address of that local to the helpers.